spec/disk_spec.rb

```diff
@@ -37,7 +37,7 @@
 
       disk = LinuxAdmin::Disk.new :path => '/dev/hda'
       allow(disk).to receive(:run!).and_return(double(:output => fdisk))
-      expect(disk.size).to eq(500.1.gigabytes)
+      expect(disk.size).to eq(500107862016)
```

Comment:
Exactly.  We were incorrectly converting to bytes when we were just given the value in bytes to start with so we didn't need to be converting at all.